bezsmertnyi-stanislav/Readme.MD

```diff
@@ -0,0 +1,68 @@
+Flashscore Live Scores
+======================
+
+## Introduction
+
+The Flashscore Live Scores app provides real-time sports scores and updates from various sporting events worldwide. Stay up-to-date with the latest scores, fixtures, and results for your favorite sports and teams. The most up-to-date live scores, results, stats, news and other info from football, tennis, basketball and 30+ sports and more than 6000 competitions from all around the world. You can select your favourite matches, teams or players, and our notifications will let you know about every important moment of the match.
+
+![](https://i.imgur.com/NRPPEeA.jpg)
+
+## Installation
+
+* Download the Flashscore Live Scores app from [AppStore](https://apps.apple.com/app/id766443283?mt=8) or [Google Play Store](https://play.google.com/store/apps/details?id=eu.livesport.FlashScore_com)
+* Install the app. You'll find the downloaded app in "Notifications" or in the "Download" folder. Click on it, choose "Install" and confirm
+* Launch the app
+
+## Features
```

Comment:
Notes on the features list:
* I understand why you decided to capitalize the main features, but it's generally not recommended. See https://learn.microsoft.com/en-us/style-guide/capitalization ("Don't use all uppercase for emphasis.")
* When listing anything, try to apply the parallelism principle -- that is, each list item should follow the same grammatical structure. For example, if you start your list with an imperative (Discover live scores...), make the rest of the items use imperative as well (Receive updates... Follow your favorite matches... Select your preferred... and so on). Or if you go with a "You can do something..." phrase, use that in other items too.